identity/src/issuers.ts

```diff
@@ -1,32 +1,39 @@
 import * as DIDKit from "@spruceid/didkit-wasm-node";
+import { NullifierGenerators } from "./credentials.js";
+import { getKeyVersions } from "./keyManager.js";
+import { HashNullifierGenerator } from "./nullifierGenerators.js";
 
 const key = process.env.IAM_JWK;
 const __issuer = DIDKit.keyToDID("key", key);
 const eip712Key = process.env.IAM_JWK_EIP712;
 const __eip712Issuer = DIDKit.keyToDID("ethr", eip712Key);
 
-const validIssuers = new Set([__issuer, __eip712Issuer]);
-
-export function getEd25519IssuerKey(): string {
-  return key;
-}
-
-export function getEd25519Issuer(): string {
-  return __issuer;
-}
-
-export function getEip712IssuerKey(): string {
-  return eip712Key;
+export function getIssuerInfo(signatureType: string): {
+  issuerKey: string;
+  nullifierGenerators: NullifierGenerators;
+} {
+  if (signatureType === "EIP712") {
+    const { activeKeyVersions, issuerKeyVersion } = getKeyVersions();
+
+    return {
+      issuerKey: issuerKeyVersion.key,
+      nullifierGenerators: activeKeyVersions.map(({ key, version }) =>
+        HashNullifierGenerator({ key, version })
+      ) as NullifierGenerators,
+    };
+  } else {
+    return {
+      issuerKey: key,
+      nullifierGenerators: [HashNullifierGenerator({ key, version: "0.0.0" })],
+    };
+  }
 }
 
-export function getEip712Issuer(): string {
-  return __eip712Issuer;
-}
+export function hasValidIssuer(issuer: string): boolean {
+  const { initiatedKeyVersions } = getKeyVersions();
+  const initiatedIssuers = initiatedKeyVersions.map(({ key }) => DIDKit.keyToDID("key", key));
 
-export function getIssuerKey(signatureType: string): string {
-  return signatureType === "EIP712" ? eip712Key : key;
-}
+  const validIssuers = new Set([__issuer, __eip712Issuer, ...initiatedIssuers]);
```

Comment:
I think that we should:
- drop `__issuer` completely
- incorporate  `__eip712Issuer` in initiatedIssuers